code/game/objects/items/weapons/twohanded.dm

```diff
@@ -373,7 +373,8 @@
 		if(!isliving(target))
 			return
 		else
-			target.Weaken(4)
+			if(!istype(src, /obj/item/weapon/twohanded/chainsaw/rubber))

```

Comment:
Ehhhh I'd rather just insert the attack proc override.
Wouldn't I have to re-define dual wielding in the toys.dm anyways?